<launch>
    <!-- 启动generatePointCloud节点 -->
  <node pkg="map" type="generatePointCloud" name="depth_to_pointcloud_node" output="screen">
    <!-- 设置参数，如果需要 -->
    <param name="camera_factor" value="1000" />
    <param name="camera_cx" value="160" />
    <param name="camera_cy" value="120" />
    <param name="camera_fx" value="120.00000000000001" />
    <param name="camera_fy" value="120.00000000000001" />
  </node>
</launch>
